docs/web/sdk-service-reference.mdx

```diff
@@ -0,0 +1,239 @@
+---
+title: Service Reference
+description: Geocortex Web - Service Reference
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import UseCaseCard from "../../src/components/UseCaseCard";
+import UseCaseContainer from "../../src/components/UseCaseContainer";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+Services act as a repository of accessible behavior and data, and can be used for a variety of purposes, from implementing commands and operations to managing shared data, to interfacing with the REST API of an external service.
+
+## Service Registration
+
+All services need to be registered with Geocortex Web. Services can optionally use the `itemType` property to bind to an app item in config.
```

Comment:
```suggestion
All services need to be registered with Geocortex Web. Services can optionally use the `itemType` property to bind to an item in [app config](concepts-app-config.mdx).
```

Reply:
Changed to link to the "app items" section of the app config docs

Reply:
Even better!